raspiraw.c

```diff
@@ -636,6 +755,35 @@ static int parse_cmdline(int argc, const char **argv, RASPIRAW_PARAMS_T *cfg)
 	return 0;
 }
 
+int reg_index(struct mode_def *sensor_mode, int r)
+{
+	int i;
+	for(i=sensor_mode->num_regs-1; i>=0; --i)
+	{
+		if (sensor_mode->regs[i].reg == r)
+			break;
+	}
+	if (i<0)
+	{
+		vcos_log_error("register %04x not found - aborting", r);
+		exit(-2);
+	}
+	return i;
+}
+
+//The process first loads the cleaned up dump of the registers
```

Comment:
Move right to the top of the file where the other enums and prototypes are (eg line 235)